Sources/SwiftDocC/Semantics/Metadata/DisplayName.swift

```diff
@@ -26,6 +26,7 @@ import Markdown
 /// }
 /// ```
 public final class DisplayName: Semantic, AutomaticDirectiveConvertible {
+    public static let introducedVersion = "5.6"
```

Comment:
I'm having trouble confirming but I think this _might_ have been 5.7?

Reply:
It looks like you're right. I was only looking at when each version was released but this doesn't seem to be on the 5.6 branch.